doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -0,0 +1,207 @@
+(kubectl-plugin)=
+
+# Use Kubectl Plugin (Alpha Feature)
+
+Today it is incredibly challenging for data scientists and AI researchers unfamiliar with Kubernetes to start using Ray on Kubernetes. However, running Ray on Kubernetes is advantageous or necessary in certain environments.
+
+Therefore, starting from KubeRay v1.2.2, we have introduced a `kubectl ray` plugin to simplify the process of running Ray on Kubernetes. The plugin is designed to help users who are not familiar with Kubernetes to run Ray on Kubernetes with ease.
+
+## Installation
+
+Please follow the instructions in the [KubeRay Kubectl Plugin](https://github.com/ray-project/kuberay/tree/master/kubectl-plugin) to install the plugin.
```

Comment:
We bundle the plugin binary into the KubeRay release now right? can we just download the bianry from the release?

Reply:
Something like:
```
curl -LO https://github.com/ray-project/kuberay/releases/download/v1.2.2/kubectl-ray_v1.2.2_linux_amd64
cp kubectl-ray_v1.2.2_linux_amd64 /usr/local/bin/kubectl-ray 
```

Reply:
Issue created: https://github.com/ray-project/kuberay/issues/2449

Reply:
Thanks I think we can directly add this part into this page too:
```
curl -LO https://github.com/ray-project/kuberay/releases/download/v1.2.2/kubectl-ray_v1.2.2_linux_amd64.tar.gz
tar -xvf kubectl-ray_v1.2.2_linux_amd64.tar.gz
cp kubectl-ray ~/.local/bin
```

Reply:
Done